### PR TITLE
publish and add Travis CI

### PR DESCRIPTION
**Pull request by MarcAntoine-Arnaud**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
-version = "0.1.0"
-authors = ["Anna Scholtz <anna@scholtzan.net>"]
+name = "rdf"
+version = "0.1.4"
+authors = ["Anna Scholtz <anna@scholtzan.net>", "Marc-Antoine Arnaud <arnaud.marcantoine@gmail.com>"]
+description = """
+rdf is a library for the Resource Description Framework (RDF) and SPARQL implemented in Rust.
+"""
+license = "MIT"
+repository = "https://github.com/media-io/rdf-rs"
```
Comment (scholtzan) [inline Cargo.toml]: I would prefer if this is the link to the main repo.

**Review by scholtzan — CHANGES_REQUESTED** [review]
Thanks for fixing the formatting issues! :) 
Sorry for my (very) delayed review. I was quite busy the past few months but am planning to get back to that project (add SPARQL support, do some cleanup, ...).

I also saw that you published your fork on [crates.io](https://crates.io/crates/rdf). I think it might more sense to use the main repository there for publishing and add me as an owner there.

**Comment by MarcAntoine-Arnaud** [comment]
Hi Anna !

Yes I'm agreed it can be better to add yourself too on the repository.
But right now I don't have your username on crates.io.

File: Cargo.toml (hunk 1 above)
Comment (MarcAntoine-Arnaud) [reply Cargo.toml]: Agreed if you are more active now it can be awesome !

**Comment by scholtzan** [comment]
It should be the same as my Github user name: scholtzan

**Comment by MarcAntoine-Arnaud** [comment]
I have invited you. I you can check.

**Pull request merged**